index.js

```diff
@@ -344,14 +349,17 @@ RedisClient.prototype.on_ready = function () {
                 self.emit("ready");
             }
         };
-        Object.keys(this.subscription_set).forEach(function (key) {
-            var parts = key.split(" ");
-            if (exports.debug_mode) {
-                console.warn("sending pub/sub on_ready " + parts[0] + ", " + parts[1]);
-            }
-            callback_count++;
-            self.send_command(parts[0] + "scribe", [parts[1]], callback);
-        });
+
+        if(this.enable_resubscribing_on_ready){
```

Comment:
Writing this as:
```js
if (this.options.disable_resubscribing_on_ready) {
    return;
}
```
should be sufficient. No more lines should be needed for this.